web_dashboard_tile/models/tile_tile.py

```diff
@@ -0,0 +1,276 @@
+# -*- coding: utf-8 -*-
+# © 2010-2013 OpenERP s.a. (<http://openerp.com>).
+# © 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
+# © 2015-Today GRAP
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+import datetime
+import time
+from dateutil.relativedelta import relativedelta
+from collections import OrderedDict
+
+from odoo import api, fields, models
+from odoo.tools.safe_eval import safe_eval as eval
+from odoo.tools.translate import _
+from odoo.exceptions import ValidationError, except_orm
+
+
+def median(vals):
+    # https://docs.python.org/3/library/statistics.html#statistics.median
+    # TODO : refactor, using statistics.median when Odoo will be available
+    #  in Python 3.4
```

Comment:
The version of Odoo we migrate this module to is already on python `>=3.5` so remove this function and replace it with `statistics.median`